acquisitions/src/main/resources/thunderjet/mod-orders/features/bind-piece.feature

```diff
@@ -659,3 +662,126 @@ Feature: Verify Bind Piece feature
     When method GET
     Then status 200
     And match response.itemId == prevItemId2
+
+
+  @Positive
+  Scenario: Bind pieces and remove binding one by one, verify piece bindItemId and isBound attributes, verify title bindItemIds
+    * def pieceId1 = call uuid
+    * def pieceId2 = call uuid
+    * def titleId2 = call uuid
+
+
+    # 1. Creating Title
+    * table titleDetails
+      | titleId  | poLineId  |
+      | titleId2 | poLineId2 |
+    * call createTitle titleDetails
```

Comment:
Do you need the response or can it be prefixed with `def v = .. ` ?

Reply:
Thanks, added